src/Symfony/Component/Serializer/Mapping/AttributeMetadata.php

```diff
@@ -127,17 +127,45 @@ public function getMaxDepth(): ?int
     /**
      * {@inheritdoc}
      */
-    public function setSerializedName(string $serializedName = null)
```

Comment:
Public methods of non-internal classes cannot be removed, this will be a BC break